nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/pubsub/PublishGCPubSub.java

```diff
@@ -137,9 +211,10 @@ public Set<Relationship> getRelationships() {
     @OnScheduled
     public void onScheduled(ProcessContext context) {
         try {
+            storedException.set(null);

```

Comment:
The `storedException` usage should be removed and failures should be thrown in this method, instead of allowing the framework to start the Processor.